{"status": "in-progress", "priority": "bug", "title": "Github model in oh-mainline:mysite/customs/models.py: github_name should not be unique=True", "milestone": "", "superceder": "", "nosylist": ["paulproteus, soonick5"], "assigned": "soonick5", "waitingon": "", "keywords": [], "id": "800", "files": [], "history": [{"message": "Looks like <a href=\"http://openhatch.org/bugs/issue802\">http://openhatch.org/bugs/issue802</a> took care of this?\n\nAdrian, if that's the case, would you close this bug please? I didn't want to summarily close it \nwithout checking with you.\n   \n", "author": "jwm"}, {"message": "Hello Asheesh,\n\nCan you please take a look at the pull request and let me know if you have any\ncomments?\n\nThanks,\nAdrian\n   \n", "author": "soonick5"}, {"message": "A screenshot of the bug: <a href=\"http://imgur.com/BZL83\">http://imgur.com/BZL83</a>\n   \n", "author": "paulproteus"}, {"message": "Right now, if you have a community on Github with a single Github username and a \nbunch of repositories with Github issue trackers, you may only add one to \nOpenHatch because github_name has unique=True in mysite/customs/models.py.\n\nThis is totally unnecessary and we should remove that restriction.\n   \n", "author": "paulproteus"}]}